.gitlab/pipeline/build.yml

```diff
@@ -139,7 +139,7 @@ build-short-benchmark:
     - .run-immediately
     - .collect-artifacts
   script:
-    - cargo build --profile release --locked --features=runtime-benchmarks
+    - cargo build --profile release --locked --features=runtime-benchmarks --bin polkadot
     - mkdir -p artifacts
     - target/release/polkadot --version
     - cp ./target/release/polkadot ./artifacts/
```

Comment:
There is also `mv ./target/testnet/polkadot ./artifacts/.` in the step `build-linux-stable`, so this step overrides that, right?
Possibly we could do something like: `cp ./target/release/polkadot ./artifacts/short-benchmarks/`.

Also the same for `polkadot-parachain` binary [build-linux-stable-cumulus](https://github.com/paritytech/polkadot-sdk/blob/master/.gitlab/pipeline/build.yml#L165) does `mv ./target/release/polkadot-parachain ./artifacts/.` but also `build-short-benchmark-cumulus` does `cp ./target/release/polkadot-parachain ./artifacts/.

I am not missing anything, there is a possible consequence that later jobs could use binary with `--features runtime-benchmarks`.